File: app/src/main/java/com/zy/ppmusic/callback/TimeTikCallBack.java
package com.zy.ppmusic.callback;

/**
 * @author ZhiTouPC
 * @date 2018/1/19
 */

public interface TimeTikCallBack {
    /**
     * 更新时间
     * @param mis 剩余时间（毫秒）
     */
    void onTik(long mis);
}

File: app/src/main/java/com/zy/ppmusic/callback/TimeTikCallBack.kt
package com.zy.ppmusic.callback

/**
 * @author ZhiTouPC
 * @date 2018/1/19
 */

interface TimeTikCallBack {
    /**
     * 更新时间
     * @param mis 剩余时间（毫秒）
     */
    fun onTik(mis: Long)
}
